### DISCUSSION TITLE
Brand new install leaves erroneous config file

### DISCUSSION THREAD

**Discussion by marcelgsteiger** [rern/rAudio#1031] [Bug Reports]
Hello all
This is my first attempt to install archlinux and rOS (and my first experience with these linux flavours), I exactly followed the "custom build" instructions (RPI4, SD Card only, WIFI setup). I started with only the modules I will eventually need: chromium, DSP, uPNP (no bluetooth etc.). The installation process (starting on a ubuntu 20.04LTS system and continuing after SSH login to the newly created box) terminated flawlessly - only giving a message
```
Failed to disable unit: Unit file bluetooth.service does not exist.
```
at the end which hopefully can be ignored, followed by
```
rAudio 1 created successfully. Reboot ...
```

After reboot I can successfully ssh to the new box (root user), but when I surf to the IP of my newly created box, my webbrowser replies 
```
Error: JSON.parse: unexpected non-whitespace character after JSON data at line 37 column 1 of the JSON data
```
without any indication what file is the problem (see screenshot below). I believe the install script left a config file in a inconsistent state. The box is currently headless (only SSH access, I'm waiting for delivery of the display), so if there is a way to fix this or update the system to the newest state via ssh root login, I would like to learn how. The doc tells "Before setup anything: Settings > Addons > rAudio > Update (if available)" ... this might perhaps help, but I do not know how to do this via ssh command line.

Any ideas?
Thank you for help and best regards
Marcel

![screenshot](https://user-images.githubusercontent.com/11504328/190912418-35ab9351-1fa0-4194-8cb7-ecccab03890b.png)

**Comment by marcelgsteiger** [comment]
..using the raudio android app, I get the same result, just without a line number and another position (position 874), the file might have been transmitted without line ends.

**Comment by marcelgsteiger** [comment]
I repeated the install twice from scratch (Version 20220917 64bit for RPI4), first with all components enabled, then again without activating WLAN on boot. The same happens. To me it seems that someting after the first closing curly brace is wrong, or the curly brace between "volumenone" : false and "repeat" : false is an error.

**Comment by rern** [comment]
@marcelgsteiger,

Please try a pre-built image file from [here](https://github.com/rern/rAudio-1/releases/tag/i20220904).

**Reply by marcelgsteiger** [reply]
Thank you, this worked perfectly. I will perhaps try later again when I have learned more about this excellent system.

**Reply by marcelgsteiger** [reply]
...hovever after a few hours of observing the same error message appears again from time to time, while the system continues to play. I currently have no media database yet, just am listening to webradio using the pre-built image. The problem can disappear from time to time, supposedly when the played track changes. Presumably it's dependent of the content to be displayed on the webpage.

![Bildschirmfoto vom 2022-09-20 12-34-50](https://user-images.githubusercontent.com/11504328/191236426-37c846b7-f395-4263-9b51-6753b33c71b2.png)
